string name
sensor_msgs/Image mask